package.json

```diff
@@ -15,25 +15,25 @@
     "lint:ts": "eslint . --ext .js,.jsx,.ts,.tsx --color",
     "lint:ts:fix": "npm run lint:ts -- --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
-    "nightly": "npm run deploy",
```

Comment:
Don't think we want to remove scripts they added @karelhala ...

Reply:
This is inherited from `starter-app` we are no longer doing these builds.